package/yast2.changes

```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Tue Jun  2 11:26:50 UTC 2015 - jreidinger@suse.com
+
+- reduce count of extending snapper for snapshotting (fate#317973)
+- 3.1.128
```

Comment:
The changelog is completely misleading: You are not extending snapper. Instead you extend the instsys with snapper.